CMakeLists.txt

```diff
@@ -34,24 +34,30 @@ set(IODE_BINARY_DIR ${CMAKE_BINARY_DIR})
 # -- resources directory --
 set(IODE_RESOUCES_DIR "${PROJECT_SOURCE_DIR}/resources")
 
+# -- compiler options --
+# SET(CMAKE_CXX_FLAGS_DEBUG "")
+# SET(CMAKE_CXX_FLAGS_RELEASE "/O2 /Ob2 /Oi /Ot")
```

Comment:
@jmpplan @gbryon 

It is possible to add specific compiler flags to all targets using the CMake commands
```
SET(CMAKE_CXX_FLAGS_DEBUG "list of commands")
SET(CMAKE_CXX_FLAGS_RELEASE "list of commands")
```